extra/2-piping.js

```diff
@@ -37,7 +37,9 @@ let badCode = format(startingValue);
 
 /* BETTER PRACTICE */
 
-let goodCode = format(startingValue);
+let goodCode = format(24);
+let goodCode1 = add(10, 2);
+let goodCode2 = multiply(10, 2);
```

Comment:
You should be starting with the value of 2, then adding 10, then multiplying by 2, then formatting - the result of each function should be passed to the next function.